app/controllers/twilio_controller.rb

```diff
@@ -2,12 +2,14 @@
 require 'sanitize'
 require 'csv'
 
+TWILIO_API_HOST = 'api.twilio.com'
```

Comment:
Seems like `TWILIO_API_HOST` is not used. Could you please double check, if this is the case, please remove it. We don't want to leave dead code. 😉